src/components/invoice/EditInvoiceViewModel.js

```diff
@@ -26,7 +25,7 @@ export default class EditInvoiceViewModel {
     this.number = obj.number || 1;
     this.client = obj.client;
     this.your = obj.company || config.company;
-    this.date = obj.date || moment(); //.subtract(1, 'months').endOf('month');
+    this.date = obj.date || new Date(); //.subtract(1, 'months').endOf('month');
```

Comment:
Waarom is deze wijziging er?
Is dit voor een bepaalde reden?
Zou dit problemen kunnen geven?

`const month = invoice.date.format('YYYYMM');`
Als de new Date() ooit uitgevoerd wordt, dan zal het op bovenstaande code crashen.